media_manager/lti.py

```diff
@@ -19,7 +20,7 @@ def _check_request(self):
         elif 'LTI_LAUNCH' not in self.request.session:
             raise LTILaunchError("Session is missing LTI_LAUNCH dict.")
         else:
-            session_keys = self.request.session['LTI_LAUNCH'].keys()
+            session_keys = list(self.request.session['LTI_LAUNCH'].keys())
```

Comment:
I'm curious why you casted this to a list here?

Reply:
Great question. This is because, while the return type of `somedictionary.keys()` is `list` in python 2, it is `dict_keys` in python 3. Accordingly, to keep the return type consistent regardless of whether one is using python 2 or 3, one must cast to `list`.

At any rate, this particular code block can do without the casting since the only critical operation the result `session_keys` is subjected to is `len(session_keys)`, which will work fine without casting. But because the code is supposed to be python 2/3 compatible (as opposed to only python 3 compatible), it might be good to have the return type consistent, even though nothing will break without the casting.